docs/ads.md

```diff
@@ -0,0 +1,458 @@
+
+#
+
+> ### <a name="AD_MANIFEST_LOADED"></a>AD_MANIFEST_LOADED
+>
+> Fires when the ad manifest has been loaded and the ad layout is available.
+> <br><br>_payload parameters:_
+>
+> | Name               | Type            | Description                                                      |
+> | ------------------ | --------------- | ---------------------------------------------------------------- |
+> | `adBreaksPosition` | `Array<number>` | An array that represents the ad break's time on the media timeline. |
+
+#
+
+> ### <a name="AD_LOADED"></a>AD_LOADED
+>
+> Fires when an ad has been loaded and ad data is available.
+> <br><br>_payload parameters:_
+>
+> | Name | Type        | Description  |
+> | ---- | ----------- | ------------ |
+> | `ad` | [`Ad`](#Ad) | The ad data |
+
+#
+
+> ### <a name="AD_STARTED"></a>AD_STARTED
+>
+> Fires when an ad has been started and ad data is available.
+> <br><br>_payload parameters:_
+>
+> | Name | Type        | Description  |
+> | ---- | ----------- | ------------ |
+> | `ad` | [`Ad`](#Ad) | The ad data |
+
+#
+
+> ### <a name="AD_RESUMED"></a>AD_RESUMED
+>
+> Fires when the ad resumed after a pause.
+
+#
+
+> ### <a name="AD_PAUSED"></a>AD_PAUSED
+>
+> Fires when the ad paused.
+
+#
+
+> ### <a name="AD_CLICKED"></a>AD_CLICKED
+>
+> Fires when the ad has been clicked by the user.
+
+#
+
+> ### <a name="AD_SKIPPED"></a>AD_SKIPPED
+>
+> Fires when the ad has been skipped.
+
+#
+
+> ### <a name="AD_COMPLETED"></a>AD_COMPLETED
+>
+> Fires when the ad completes playing.
+
+#
+
+> ### <a name="AD_ERROR"></a>AD_ERROR
+>
+> Fires when an error occurred while the ad was loading or playing.\
+> <br><br>_payload parameters:_
+>
+> | Name       | Type     | Description                                                                 |
+> | ---------- | -------- | --------------------------------------------------------------------------- |
+> | `severity` | `string` | The error severity                                                    |
+> | `category` | `string` | The error category                                                        |
+> | `code`     | `number` | The error code                                                             |
+> | `data`     | `Object` | The error data: the original ad error can be accessed by `data.innerError`. |
+
+#
+
+> ### <a name="ALL_ADS_COMPLETED"></a>ALL_ADS_COMPLETED
+>
+> Fires when the player is done playing all the ads.
+
+#
+
+> ### <a name="AD_BREAK_START"></a>AD_BREAK_START
+>
+> Fires before an ad break is about to start.
+> <br><br>_payload parameters:_
+>
+> | Name      | Type                  | Description        |
+> | --------- | --------------------- | ------------------ |
+> | `adBreaK` | [`AdBreak`](#AdBreak) | The ad break data |
+
+#
+
+> ### <a name="AD_BREAK_END"></a>AD_BREAK_END
+>
+> Fires when an ad break is finished.
+
+#
+
+> ### <a name="AD_FIRST_QUARTILE"></a>AD_FIRST_QUARTILE
+>
+> Fires when the ad playhead crosses first the quartile.
```

Comment:
Just verifying that "ad playhead" is a term that's use in the industry; I saw that BrightCove also use it but just checking...

Reply:
@AnnaAleksandrowicz yes, this is a coined term